tests/SprykerEcoTest/Zed/AmazonPay/_support/_generated/AmazonPayBusinessTesterActions.php

```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] d5e5a2fcade9574c1e5fe52dbf180464
+<?php  //[STAMP] 3ff3b12b97b37e0e8e89ed2a3bb65fbd
```

Comment:
Why this is in changes. It should be under the gitignore.

Reply:
It wasn't added by me